bindings/nodejs/lib/types/wallet/address.ts

```diff
@@ -60,11 +60,3 @@ export interface SendNftParams {
     /** The ID of the NFT to send. */
     nftId: NftId;
 }
-
-/** Options for address generation, useful with a Ledger Nano SecretManager */
-export interface GenerateAddressOptions {
```

Comment:
I guess this has been removed because there was a similar one in `generate-addresses-options.ts`, but this one had no optional fields and it seemed to better match the one in Rust?

Reply:
odd, because it was added earlier, but yeah, that's the reason. I'll match it with the Rust one.